tests/functional_tests/autolab.sh

```diff
@@ -19,6 +19,13 @@ export TESTDIR
 # install node dependencies
 npm --quiet install 1>/dev/null 2>&1
 
+# Run the tests in the test_modules directory
```

Comment:
It is better to move the initialisation work of `test_modules` to `tests/test_modules/init.sh` and invoke the script in `tests/test.sh`